tests/kafkatest/services/kafka/kafka.py

```diff
@@ -89,11 +91,38 @@ def start(self):
                 topic_cfg["topic"] = topic
                 self.create_topic(topic_cfg)
 
+    def set_protocol_and_port(self, node):
```

Comment:
@benstopford  @ijuma  There were no tests at the time which needed separate ports, so I was just keeping the code simple. Geoff's suggestion looks good.